test/tag_invoke.cpp

```diff
@@ -0,0 +1,165 @@
+// test/call_result_t.cpp                                             -*-C++-*-
```

Comment:
The files do contain their respective filename to potentially make sense out of context. If we don't want that, I can remove them. If we want to retain that, I think the name should be adjusted.

Reply:
Copy & paste error here